c/camel-quarkus/camel-quarkus_ubi_8.5.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash -e
+# ----------------------------------------------------------------------------
+#
+# Package	    : camel-quarkus
+# Version	    : main
+# Source repo	: https://github.com/apache/camel-quarkus.git
+# Tested on	    : UBI 8.5
+# Language      : java
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer	: Shreya Kajbaje <Shreya.Kajbaje@ibm.com>
+#
+# Disclaimer: This script has been tested in non-root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+# Install dependencies
+yum -y update && yum install -y git wget java-17-openjdk-devel tar
+JDK_PATHS=$(compgen -G '/usr/lib/jvm/java-17-openjdk-*')
+export JAVA_HOME=${JDK_PATHS%$'\n'*}
+export PATH=$JAVA_HOME/bin:$PATH
+
+# Install maven
+wget http://archive.apache.org/dist/maven/maven-3/3.8.4/binaries/apache-maven-3.8.4-bin.tar.gz
+tar -zxf apache-maven-3.8.4-bin.tar.gz
+cp -R apache-maven-3.8.4 /usr/local
+ln -s /usr/local/apache-maven-3.8.4/bin/mvn /usr/bin/mvn
+
+#-------------------------------Camel Build & Tests-------------------------------
+
+PACKAGE_NAME=camel
+PACKAGE_VERSION=${1:-camel-4.2.0}
```

Comment:
except `camel-quarkus` clone, build steps.. for other 2 dependency build ( camel & camel-sb) you may not need to put PACKAGE_NAME and PACKAGE_VERSION variable.
If you do may be command line `PACKAGE_VERSION=${1:-camel-4.2.0}` provision not needes .. just directly pass version you are building

Reply:
@nikhil-kalbande  - we would like to keep both package name and version since there are multiple GH repos which are invoked in this build script via `PACKAGE_URL`. Is that going to hinder the currency back end workflow if we decide to keep both - `PACKAGE_NAME and PACKAGE_VERSION` ?

Reply:
Should not hinder actually @ghatwala, ok to keep variables. 
May just need to remove command line `${1:-camel-4.2.0}` provision for other repos, as we pass version through as param to build script while executing in pipeline. So in current code it will consider same version for all github repo you have mentioned in build script and it will break build

Reply:
Sure Nikhil , above should be possible , @shreyakajbaje requesting you to please check here.

Reply:
@shreyakajbaje @ghatwala although we are building multiple packages in the script, aren't they all dependencies for camel-quakus? I think its okay to do that part as we are simply installing from source instead of lets say the distro or some other source. 

However, I am not sure I understand why we want a *different* package_name more so since "camel4-csb-ceq-camel-k" is not going to make any sense to an end user who comes to lets say find something on the currency portal. @nikhil-kalbande  correct me if I am wrong here. Besides it also breaks the naming conventions we follow elsewhere in this repository

Reply:
so @seth-priya @nikhil-kalbande   - above ask was from IBM/Amir to have a single job for all camel upstream repos ( camel-4 , camel-springboot , camel-quarkus),  as to utilize artifacts locally instead of upstream maven which implied a single job name but if that breaks naming convention @prasanna-marathe@github.com - please convey to Amir accordingly that we will name job to maintain consistency.

Reply:
@ghatwala We will convey to Amir in our regular DSM